src/packet/bwe/acked_bitrate_estimator.rs

```diff
@@ -73,8 +73,7 @@ impl AckedBitrateEstimator {
         // current estimate. With low values of uncertainty_symmetry_cap_ we add more
         // uncertainty to increases than to decreases. For higher values we approach
         // symmetry.
-        let sample_uncertainty =
-            scale * (estimate_bps - sample_estimate_bps).abs() / (estimate_bps.max(25_000.0));
+        let sample_uncertainty = scale * (estimate_bps - sample_estimate_bps).abs() / estimate_bps;
```

Comment:
I don't see how this aligns with `bitrate_estimator.cc` better than it did before. The current code doesn't seem super aligned either, but that might be because libWebRTC has changed.

Reply:
@k0nserv I just get libwebrtc version 9f3ccf291e from comment in acked_bitrate_estimator.rs, may be the sample_uncertainty code is from older libwebrtc version

Reply:
Yes or I made some change here when porting. My point is that neither the previous code nor your change matches the libWebRTC version. If we are going to change it shouldn't we try to match libWebRTC?

Reply:
Yes, it would be best if we could match the libWebRTC implementation.

Reply:
I see the missing part is only in_alr param in update function.

Reply:
Hmm, isn't the corresponding code for this

```cpp
  float sample_uncertainty =
      scale * std::abs(bitrate_estimate_kbps_ - bitrate_sample_kbps) /
      (bitrate_estimate_kbps_ +
       std::min(bitrate_sample_kbps,
                uncertainty_symmetry_cap_.Get().kbps<float>()));
```

We don't have the `uncertainty_symmetry_cap`, but it seems to default to zero anyway.

Reply:
@k0nserv uncertainty_symmetry_cap default is zero then I think we can ignore it. The only thing I don't understand is why we need `.max(25_000.0)`. After I removed it estimated bitrate increased faster.

Reply:
Honestly I don't recall why I did `.max(25_000.0)`. However, the code is also wrong here, we divide by `estimate_bps` when it should be `sample_estimate_bps` I think if you fix that we should be okay

Reply:
I think there's something wrong. We can ignore `std::min(bitrate_sample_kbps, uncertainty_symmetry_cap_.Get().kbps<float>())` because uncertainty_symmetry_cap_ is zero, then we only need do device by `bitrate_estimate_kbps_`, which is `estimate_bps` in str0m.

Reply:
> only need do device by bitrate_estimate_kbps_, which is estimate_bps in str0m.
This should be `bitrate_sample_kbps` which is `sample_estimate_bps` in str0m. Currently we divide by `estimate_bps`. If you change that I think we are good to merge

Reply:
Can you explain why it is `bitrate_sample_kbps `? As shown in the original code, it used `bitrate_estimate_kbps_ `.

```cpp
float sample_uncertainty =
      scale * std::abs(bitrate_estimate_kbps_ - bitrate_sample_kbps) /
      (bitrate_estimate_kbps_ +
       std::min(bitrate_sample_kbps,
                uncertainty_symmetry_cap_.Get().kbps<float>()));
```

Reply:
Sorry I'm just being daft, I think this change is fine let's do it!

Reply:
We're ready to merge?

Reply:
Go for it!

Reply:
It says you have changes requested, should I override?